src/tests/public_api/test_helpers/proxy.h

```diff
@@ -132,6 +254,10 @@ class Proxy : public core::Thread, private packet::IWriter {
         return input_repair_endp_;
     }
 
+    const roc_endpoint* control_endpoint() const {
+        return input_control_endp_;
+    }
```

Comment:
```suggestion
    const roc_endpoint* control_endpoint() const {
        CHECK(input_control_endp_);
        return input_control_endp_;
    }
```

Let's make it fail early if trying to use control endpoint when it's not configured.